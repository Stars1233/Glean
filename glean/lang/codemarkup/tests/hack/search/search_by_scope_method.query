query: search.code.SearchByScope { name = "bar", scope = ["RefClass"] }
transform: [normord, []]
perf: true
