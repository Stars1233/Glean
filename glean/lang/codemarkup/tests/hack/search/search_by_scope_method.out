[
  "@generated",
  {
    "key": {
      "entity": {
        "hack": {
          "decl": {
            "method": {
              "key": {
                "container": {
                  "class_": {
                    "key": {
                      "name": { "key": { "name": { "key": "RefClass" } } }
                    }
                  }
                },
                "name": { "key": "bar" }
              }
            }
          }
        }
      },
      "language": 5,
      "name": "bar",
      "scope": [ "RefClass" ]
    }
  }
]